Lib/typing.py

```diff
@@ -2459,7 +2459,7 @@ def __init__(self, name, tp):
 
     def __mro_entries__(cls, bases):
         raise TypeError(
-            f'You cannot subclass an instance of NewType, perhaps you were looking for:\n'
+            f'Cannot subclass {cls.__name__}, perhaps you were looking for:\n'
```

Comment:
I also vote for a single-line version of this message, no need to use `\n`. Do you agree?

Reply:
I think it's clear and easy to copy and paste if it's two lines